<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureDetectionClass</class>
 <widget class="QDialog" name="FeatureDetectionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feature Detection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_confirm">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Img File Path:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关键点数量上限:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cookie Save Path:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="4">
       <widget class="QLineEdit" name="lineEdit_write_path"/>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QLineEdit" name="lineEdit_img_path"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Ply File Path:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_numKeyPoints"/>
      </item>
      <item row="3" column="3" colspan="2">
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>特征描述方法:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QComboBox" name="comboBox_featureName">
        <item>
         <property name="text">
          <string>ORB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KAZE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AKAZE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BRISK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIFT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SURF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BINBOOST</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VGG</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="4">
       <widget class="QLineEdit" name="lineEdit_ply_read_path"/>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Intrinsic</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="2">
          <widget class="QLineEdit" name="lineEdit_fx"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>fx:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>fy:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>cy:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="lineEdit_fy"/>
         </item>
         <item row="5" column="4">
          <widget class="QLineEdit" name="lineEdit_cy"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>cx:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="lineEdit_cx"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_confirm</sender>
   <signal>clicked()</signal>
   <receiver>FeatureDetectionClass</receiver>
   <slot>onPush_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>FeatureDetectionClass</receiver>
   <slot>onPush_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPush_confirm()</slot>
  <slot>onPush_cancel()</slot>
 </slots>
</ui>
